lib/assertions.ts

```diff
@@ -1190,7 +1210,7 @@ export default class DOMAssertions {
   /**
    * @private
    */
-  private get targetDescription() {
+  private get targetDescription(): string {
```

Comment:
this might fix https://github.com/simplabs/qunit-dom/issues/550 but I'm not 100% sure.